scripts/migrations/002-remove-webview-flags.js

```diff
@@ -37,7 +52,9 @@ const removeWebViewFlags = (key, value) => {
 };
 
 /**
- * @param {string} filename
+ * Perform removal of flags within WebView data within all the data in a specified file.  The function will then automatically write any needed changes back into the file.
```

Comment:
This sounds more like the description for the previous function (`removeWebViewFlags`)?